vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/PriorityByWorkflow.java

```diff
@@ -0,0 +1,23 @@
+package ca.on.oicr.gsi.vidarr.api;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+/** Class to record priority by workflow */
+public class PriorityByWorkflow {
```

Comment:
Where is this class used?

Reply:
Also, there is also a PriorityByWorkflow in vidarr-server, which is much more fleshed out, what is the relationship between the two?